logbook-ktor-client/src/main/kotlin/org/zalando/logbook/client/LogbookClient.kt

```diff
@@ -56,7 +56,7 @@ class LogbookClient(
 
             scope.receivePipeline.intercept(HttpReceivePipeline.After) { httpResponse ->
                 val (loggingContent, responseContent) = httpResponse.rawContent.split(httpResponse)
-                scope.launch(coroutineContext) {
+                scope.launch {
```

Comment:
I'm not sure how to test this with a unit test for now. If anyone has ideas, please suggest a way.